packages/api/es/esFileSearchQueue.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const flatten = require('lodash/flatten');
+const Logger = require('@cumulus/logger');
+const { Search, defaultIndexAlias } = require('./search');
+
+const log = new Logger({ sender: '@api/es/esFileSearchQueue' });
+
+const defaultESScrollSize = 1000;
+const defaultESScrollDuration = '30s';
+
+const logAndToss = (error) => {
+  log.error(JSON.stringify(error));
```

Comment:
@cumulus/Logger loses a lot of information from esClient errors, this guarantees they are all displayed.